api/v1/mapping/src/commonMain/kotlin/Mappings.kt

```diff
@@ -878,7 +882,31 @@ fun PackageRunData.mapToApi() = ApiPackage(
     pkg.vcsProcessed.mapToApi(),
     pkg.isMetadataOnly,
     pkg.isModified,
-    shortestDependencyPaths.map { it.mapToApi() }
+    shortestDependencyPaths.map { it.mapToApi() },
+    curations.map { it.mapToApi() }
+)
+
+fun PackageCurationData.mapToApi() = ApiPackageCurationData(
+comment = comment,
+    purl = purl,
+    cpe = cpe,
+    authors = authors,
+    concludedLicense = concludedLicense,
+    description = description,
+    homepageUrl = homepageUrl,
+    binaryArtifact = binaryArtifact?.mapToApi(),
+    sourceArtifact = sourceArtifact?.mapToApi(),
+    vcs = vcs?.mapToApi(),
+    isMetadataOnly = isMetadataOnly,
+    isModified = isModified,
+    declaredLicenseMapping = declaredLicenseMapping
```

Comment:
This lacks the `sourceCodeOrigins` and `labels` properties.

Reply:
Both of them have not been implemented in the server after they were introduced in ORT, so currently they are simply ignored.

Reply:
I've created https://github.com/eclipse-apoapsis/ort-server/issues/2773 to track that.